src/app/hooks/useMediaQuery/index.test.jsx

```diff
@@ -36,9 +40,8 @@ describe('useMediaQuery', () => {
   it('should call removeListener on cleanup', () => {
     const testFn = jest.fn();
     renderHook(() => useMediaQuery('(max-width: 600px)', testFn));
-    cleanup().then(() => {
-      expect(mockRemoveListener.mock.calls.length).toEqual(1);
-      expect(mockRemoveListener.mock.calls[0][0]).toEqual(testFn);
-    });
+    cleanup();
```

Comment:
```suggestion
    await cleanup();
```
Does cleanup return a promise, just wondered if you've intentionally changed this logic?

Reply:
It doesn't return a promise from the `@testing-react/react` library, but did in the now 'old' `@testing-react/react-hooks` library